src/test/resources/org/rust/ide/intentions/fixtures/unwrap_lambda_expr/unwrap_braces.rs

```diff
@@ -0,0 +1,3 @@
+fn main () {
+    |x| { x *<caret> x }
```

Comment:
I think enabling this inspection here may be too much invasive. What do you think about limiting it only to  top-level expr (in other words, enabling only on ends of expression)